src/rng/Rnd_Control_Inline.hh

```diff
@@ -11,6 +11,7 @@
 #define rtt_rng_Rnd_Control_Inline_hh
 
 #include "Counter_RNG.hh"
+#include "device/config.h"
```

Comment:
FYI - This include is not required.  `Counter_RNG.hh` has already included `device/config.h`.

Reply:
Thanks! Removing...